src/VisualStudioEditorsSetup/VisualStudioEditorsSetup.csproj

```diff
@@ -24,6 +24,8 @@
     <NoDocumentationFile>true</NoDocumentationFile>
     <UseCommonOutputDirectory>true</UseCommonOutputDirectory>
     <ProjectSystemLayer>VisualStudio</ProjectSystemLayer>
+    <NeedsSigningToRelease>true</NeedsSigningToRelease>
+    <ProducingSignedVsix>true</ProducingSignedVsix>
```

Comment:
:) At this point, it is either marking the shipping binaries or marking the non-shipping binaries